### ISSUE TITLE
Running ./gradlew tasks produces failure

### ISSUE THREAD

**Issue by thisisalexandercook** [eisop/checker-framework#1257]
When running `./gradlew tasks` the following error occurs:

```bash
* What went wrong:
Execution failed for task ':tasks'.
> Could not create task ':checker:ainferTestCheckerGenerateJaifsInsertAnnotations'.
   > Could not get unknown property 'afu' for task ':checker:ainferTestCheckerGenerateJaifsInsertAnnotations' of type InsertAnnotationsToSource.
```
It appears there are some dead tasks preventing the use of some gradle tooling such as viewing the task tree. These tasks should be removed if it is safe to do so.

**Comment by wmdietl** [comment]
The ainfer tasks aren't currently used, as we disabled WPI in EISOP. The tasks are only there to minimize differences with upstream.

Is there some way to keep the tasks, but disable them from appearing in the `tasks` list?

When fixing this, we should add `./gradlew tasks` and maybe the task tree output to e.g. the `misc` test script, to ensure this doesn't come back.
